docs/source/technical_tutorials/access/aws_access.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>I suggest changing this:</p><p> </p><blockquote>we convert the&nbsp;<code>"s3://"</code>&nbsp;prefix to&nbsp;<code>"/vsis3/"</code>&nbsp;using the&nbsp;<code>sub()</code>&nbsp;function.</blockquote><p> </p><p>to this:</p><p> </p><blockquote>we replace the <code>"s3://"</code> prefix with <code>"/vsis3/"</code>. </blockquote><p> </p><p>There's no need to mention the specific method/function since it's obvious from looking at the code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>